db/migrate/20200206163659_add_migrated_at_to_complete_moabs.rb

```diff
@@ -0,0 +1,5 @@
+class AddMigratedAtToCompleteMoabs < ActiveRecord::Migration[6.0]
```

Comment:
this class name seems a bit misleading, but whatever, i don't think it matters much.  if you end up adding the FK constraint suggested in another comment and you want to rename this, great, if you don't care to bother that's also totally fine w/ me.

Reply:
Oh yeah, I was flipping around on  that,  I can  update.